PWGEM/PhotonMeson/Core/CutsLibrary.cxx

```diff
@@ -605,9 +589,8 @@ DalitzEECut* o2::aod::dalitzeecuts::GetCut(const char* cutName)
 
     // for PID
     cut->SetPIDScheme(DalitzEECut::PIDSchemes::kTPChadrejORTOFreq);
-    cut->SetMinPinTOF(0.16);
     cut->SetTPCNsigmaElRange(-2, +3);
-    cut->SetTPCNsigmaPiRange(-3, +3);
+    cut->SetTPCNsigmaPiRange(-1e+10, +3);
```

Comment:
This is a pretty small value. Is this wanted?

Reply:
Yes. this is necessary to reject all pions below 3 sigma.